src/extension.ts

```diff
@@ -321,7 +321,7 @@ export function activate(context: vscode.ExtensionContext): void {
     });
     registerCommand('appService.AddCosmosDBConnection', addCosmosDBConnection);
     registerCommand('appService.RemoveCosmosDBConnection', removeCosmosDBConnection);
-    registerCommand('appService.RevealConnection', async (node: CosmosDBConnection) => await node.databaseTreeItem.reveal());
+    registerCommand('appService.RevealConnection', async (node: CosmosDBConnection) => await node.cosmosExtensionItem.reveal());
```

Comment:
Excuse me, sir, we're trying to transition our `extension.ts` file to only contain the commands.

Reply:
I didn't add any new commands - just edited existing ones :P